zanata-war/src/main/java/org/zanata/action/ProjectHome.java

```diff
@@ -1063,8 +1063,8 @@ private void ensureMutualExclusivity(
     /**
      * Update the about page to the entered value, and show a success message.
      */
-    @Restrict("#{s:hasPermission(projectHome.instance, 'update')}")
     public void updateAboutPage() {
+        identity.checkPermission(instance, "update");
```

Comment:
Maybe we should just use entity listener (e.g. pre-update) to check for permission. This will save us from adding this identity.checkPermission everywhere when it's about to update the entity. Could be something for later.